### ISSUE TITLE
Vftrace could not find any parseable symbol tables associated with your executable

### ISSUE THREAD

**Issue by sbrass** [SX-Aurora/Vftrace#6]
I'm continuing to include Vftrace into my MPI project.
So far, I included Vftrace as Meson subproject (so, I do not have to explicitly care how to link (statically or dynamically) against Vftrace).

In the current setup, `libvftraca.a` is packed directly into my executable.
However, if I run my executable I get the message `Vftrace could not find any parseable symbol tables associated with your executable`.
Any suggestions?

BTW, it would be also nice if you could provide any kind of testing infrastructure and/or example for correct application? Just to get a right feeling for a typical application.

**Comment by sbrass** [comment]
I tracked through Vftrace to find the suspecting point, i.e. when setting up the symbol table and reading in the current memory map 
```
55XX55XXX00-55XX55XXX000 r--p 00000000 00:1a 8165080 <XX>/build/test/benchmark/benchmark_1
55XX55XXX00-55XX55XXX000 r-xp 00003000 00:1a 8165080 <XX>/build/test/benchmark/benchmark_1
```
With `__VMAP_OFFSET` not defined, Vftrace expects the offset to be at 0, this is obviously not the case (for me).

As a fast fix, I've just added `-D__VMAP_OFFSET` as build option, then, it works as expected.

Btw the configuration test using `config_test.sh`, which is not part of the repository, does not work then properly...
